protokube/pkg/gossip/mesh/mesh.pb.go

```diff
@@ -30,18 +30,25 @@ limitations under the License.
 */
 package mesh
 
-import proto "github.com/golang/protobuf/proto"
-import fmt "fmt"
-import math "math"
-import _ "github.com/gogo/protobuf/gogoproto"
+import (
+	fmt "fmt"
 
-import errors "errors"
+	proto "github.com/golang/protobuf/proto"
 
-import strings "strings"
-import reflect "reflect"
-import github_com_gogo_protobuf_sortkeys "github.com/gogo/protobuf/sortkeys"
+	math "math"
 
-import io "io"
+	_ "github.com/gogo/protobuf/gogoproto"
+
+	errors "errors"
+
```

Comment:
Any reason for the blank line between each import being added?

Reply:
No reason. Not sure why it got formatted this way but I'll clean it up